subsys/matter/Kconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Support for Matter protocol in NCS is experimental
+config CHIP
+	bool "Matter (formerly Connected Home over IP) protocol stack"
```

Comment:
I think you don't even need to repeat the title and help. It will be taken from the upstream, right?

Reply:
Yes, I just repeated it for the code readability purpose, but I can either remove it.

Reply:
I'd drop "formerly CHIP" here. No need to mention it at every corner in my opinion. @LuDuda ?